### PR TITLE
startOnSeek Demo

### PR DESCRIPTION
**Pull request by waxidiotic**
This demo will... you know... demo how to use the startOnSeek ad rule in conjunction with the `starttime` config option and the browser's local storage.

### CODE REVIEW

File: demos/advanced/start-on-seek/js/main.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+			rules: {
+				startOnSeek: startOnSeekValue
+			}
+		}
+	}).on('time', function(evt) {
+		localStorage['jwplayer.starttime'] = evt.position;
+	});
+
+	startOnSeek.addEventListener('change', function(e) {
+		e.preventDefault();
```
Comment (pajong) [inline demos/advanced/start-on-seek/js/main.js]: Why do we need to `preventDafault()`?

Comment (waxidiotic) [reply demos/advanced/start-on-seek/js/main.js]: Eh, we probably don't. I'll remove it.

**Comment by kcorneli201** [comment]
Is there a way to link the referenced API call `on('time')` to the documentation in the API reference?

**Comment by waxidiotic** [comment]
> Is there a way to link the referenced API call `on('time')` to the documentation in the API reference?

Every page on the dev demo site is a template. For the API calls, it's parsed from config.json and put where you see it on the page so it would be Monica's call whether or not she wants to change it. I think it's a good idea. @monibons

**Comment by monibons** [comment]
At this time we're not adding any additional functionality to the demos as we're focusing on decoupling them from the developer site in preparation for the full rollover to readme.io. In the meantime, you can always include contextual body copy to accompany the demo where you can direct people to the documentation.

**Pull request merged**